File: dayTwo/PYTHON/level two/task_five.py
first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))
third_number = int(input("Enter third number: "))

print("The largest is", end=" ")
if first_number > second_number and first_number > third_number:
    print(first_number)

elif second_number > first_number and second_number > third_number:
    print(second_number)

else:
    print(third_number)

File: dayTwo/JAVA/levelTwo/TaskFive.java
import java.util.Scanner;

public class TaskFive {

    public static void main(String... args) {

        Scanner inputCollector = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int firstNumber = inputCollector.nextInt();

        System.out.print("Enter second number: ");
        int secondNumber = inputCollector.nextInt();

        System.out.print("Enter third number: ");
        int thirdNumber = inputCollector.nextInt();

        System.out.print("The largest number is ");
        if (firstNumber > secondNumber && firstNumber > thirdNumber)
            System.out.println(firstNumber);

        else if (secondNumber > firstNumber && secondNumber > thirdNumber)
            System.out.println(secondNumber);

        else
            System.out.println(thirdNumber);
    }
}
